packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -382,10 +380,6 @@ export class DataProxyEngine extends Engine {
       try {
         return await args.callback({ logHttpCall })
       } catch (e) {
-        this.logEmitter.emit('error', {
```

Comment:
I think we would lose the logging for non-retryable errors which are not request errors if we remove it (e.g. Data Proxy HTTP 400 errors), wouldn't we? Or is that not a problem since they'll be thrown anyway?

Reply:
They will be thrown anyway, yes. Shouldn't be a problem.